plugins/snippets/luasnip/default.nix

```diff
@@ -156,7 +166,7 @@ in
           ];
       extraConfig = [
         ''
-          require("luasnip").config.set_config(${helpers.toLuaObject cfg.extraConfig})
+          require("luasnip").config.set_config(${helpers.toLuaObject cfg.settings})
```

Comment:
Either this line or the `settings` description is wrong.

You have `.config.set_config` here, but just `.set_config` in the description.

Reply:
Since CI is happy I'm guessing the description is wrong, not the implementation?

Reply:
Is the CI happy ?

Reply:
I saw a single error that seemed related to cross-compile. I.e. likely a buildbot issues, not an issue with the PR